
#Stopping Distance
Stopping_Distance : 1
Robot_Half_Width : 0.5

